### PR TITLE
Changelog for r100

### PR DESCRIPTION
**Pull request by esbenbach**
The changelog for release 100.
Accidentally we are GAing quite a bit here, and fixing lots of minor issues.

### CODE REVIEW

File: source/changelog/r100.inc [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+Another release, this one marks our 100th anniversary for releases, since we started numbering our releases at least.
+This release has lots of minor changes, a few new features, and we are making a few things generally available as well (ie. they are out of preview).
+
+As always the backend has been rolled out entirely, while the frontend/UI clients will roll out over the next couple of days.
+
+Avtalegiro Generally Available
+==============================
+With this release the Avtalegiro integration is now generally available for everyone to use.
+
+Simply put this feature allows subscribers to register AvtaleGiro mandates in with their bank (or using an online form if the tenant provides on), and mandates can be imported into INFO-Subscription for recurring charges.
```
Comment (BrynjarS) [inline source/changelog/r100.inc:15]: Clarify: in with

File: source/changelog/r100.inc [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+* Fixed an issue with download/export of EHF files where the file would not be generated and exported but instead an empty storage reference would be generated to the caller. **Hotfixed**
+* Subscriber and Plan information should now be included on all EHF files, including enterprise plan invoices.
+* Extended the EHF handling to generalize Fee handling so it now converts to charges the same way that regular charge/invoice lines does.
+* Invoices with allowances will now be materialized with Allowances in the EHF lines.
+* Fixed an issue in the order flow, where multiple simultaneous complete requests on the same order would lead to multiple subscriptions being created with the same information.
+* Importing AvtaleGiro mandates using the OCR file would cause a complete abort in case the file included a cancellation of a mandate that was not already present. This has now been fixed and replaced with a warning.
+* AvtaleGiro and Vipps integration is now consider to be generally available. See separate section for details.
```
Comment (BrynjarS) [inline source/changelog/r100.inc:60]: Spelling: consider -> considered

File: source/changelog/r100.inc [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+If the number series has holes in it, it is effectively a broken sequence and it will only work up until the hole is encountered.
+
+The introduction of this ordering represents a breaking change in the API, but our telemetry tells us that no-one but we are using that API.
+
+The UI has not yet been updated to acknowledge the new numbering, so expect some issues when managing multiple reminder setups at the same time.
+
+Partial SAFT-T File generation
+------------------------------
+BLAH WE NOW HAVE SAFT SUPPORT!
```
Comment (BrynjarS) [inline source/changelog/r100.inc:78]: Remove "BLAH"?

**Review by BrynjarS — COMMENTED** [review]
See comments

File: source/changelog/r100.inc [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+Merchant Client
+===============
+The following new features and improvements are now available in the merchant client.
+
+* Added option to download invoice PDFs for enterprise plans.
```
Comment (havmunke) [inline source/changelog/r100.inc:84]: I don't think it's the PDF. It's details from a grid.

File: source/changelog/r100.inc [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+
+Ranked/Numbered Reminder Settings
+---------------------------------
+Reminder Settings for the dunning process now contains a rank/numbering for ordering reminder production.
+
+Previously the order would be based on the creation time, thus the oldest would always be first, and deleting it and adding a new would be evaluated second regardless of the delay count.
+The rank can be specified during creation and updates, if not specified the next available rank will be taken automatically (temporary compatibility thing).
+
+If the number series has holes in it, it is effectively a broken sequence and it will only work up until the hole is encountered.
```
[the remark this answers is not in the document]
Comment (esbenbach) [reply source/changelog/r100.inc:70]: should be "work, up until" - but regardless its redundant and ill remove it :)

**Pull request merged**